examples/spear_qcp/scenario.txt

```diff
@@ -5,6 +5,7 @@ deterministic = 0
 run_obj = runtime
 overall_obj = PAR10
 cutoff_time = 5
-wallclock-limit = 30
+wallclock-limit = 100
```

Comment:
I mainly added it to let successive halving complete a couple of iterations since it evaluates a lot more configurations (so takes some time). I will change this back if it is not necessary.